internal/store/utils_test.go

```diff
@@ -167,6 +167,15 @@ func TestKubeLabelsToPrometheusLabels(t *testing.T) {
 			expectKeys:   []string{"label__app5"},
 			expectValues: []string{"starts_with_underscore"},
 		},
+		{
+			kubeLabels: map[string]string{
+				"an":    "",
+				"order": "",
+				"test":  "",
+			},
+			expectKeys:   []string{"label_an", "label_order", "label_test"},
```

Comment:
if run multiple times against the previous code, this will fail due to non-deterministic ordering in the above map at instantiation time